app/static/style.css

```diff
@@ -10,3 +10,55 @@ header h1 {
   font-weight: 700;
   text-decoration: none;
 }
+
+/* Super simple dark theme - By Jaiden W */
+
+body.dark-theme{
+  background:#202020;
+  color:#fff!;
```

Comment:
I think we don't want pure white on pure black. If we take Material design's recommendations (not that it's gospel, but it's a good starting guideline) we'd have a background of #121212 for the very back, up to #242424 for the most accentuated background, and a text color of at most #dedede